tests/replication.rs

```diff
@@ -836,7 +836,7 @@ fn diagnostics() {
     assert_eq!(stats.mappings, 1);
     assert_eq!(stats.despawns, 1);
     assert_eq!(stats.packets, 2);
-    assert_eq!(stats.bytes, 31);
+    assert_eq!(stats.bytes, 16);
```

Comment:
Changes reduce the size of small messages a lot.
But it turns out a bit slower in serialization / deserialization.